unitylibs/core/workflow/workflow.js

```diff
@@ -316,6 +317,11 @@ class WfInitiator {
         sfList: new Set([feature]),
         psw,
       },
+      'workflow-firefly': {
+        productName: 'Firefly',
+        sfList: new Set(['text-to-mage']),
```

Comment:
Seems like a typo? Should be `text-to-image`

Reply:
Hmm. It's the same on line 312 for `workflow-ai`. Not sure why. @Ruchika4?